bentoml/_internal/utils/_formparser.py

```diff
@@ -0,0 +1,180 @@
+import binascii
```

Comment:
why the renaming?

Reply:
there's no need to use leading `_` for files like this right? it's already under `_internal`

Reply:
you are correct